composer.json

```diff
@@ -29,11 +29,11 @@
   "require": {
     "php": ">=7.2.0",
     "ext-intl": "*",
-    "atk4/core": "^2.0",
-    "atk4/dsql": "^2.0"
+    "atk4/core": "~2.0.0",
```

Comment:
@romaninsh I think you should require this in all `legacy/2.0` branches in all repos, right?